ipn/ipnlocal/local.go

```diff
@@ -4931,18 +4938,7 @@ func (dt *driveTransport) RoundTrip(req *http.Request) (resp *http.Response, err
 		}
 	}()
 
-	// dialTimeout is fairly aggressive to avoid hangs on contacting offline or
-	// unreachable hosts.
-	dialTimeout := 1 * time.Second // TODO(oxtoacart): tune this
```

Comment:
Note - I took out this timeout because we now hide offline peers anyway, so this may not be much of a problem, and it was never clear whether 1 second was too aggressive anyway.